Watchman.Discord/Areas/Administration/Controllers/MutedUsersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.Administration.BotCommands;
+using Watchman.Discord.Areas.Protection.Services;
+using Watchman.DomainModel.Messages;
+
+namespace Watchman.Discord.Areas.Administration.Controllers
+{
+    public class MutedUsersController : IController
```

Comment:
w sumie nie wiem czy nie lepiej to wrzucić w MuteController
@Marcin99b co myślisz?